api/base/settings/defaults.py

```diff
@@ -284,3 +284,16 @@
 
 # If set to True, automated tests with extra queries will fail.
 NPLUSONE_RAISE = False
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'osf_cache_table',
+    },
+    'waffle_cache': {
```

Comment:
Does this change the behavior in waffle in any way? Or was this added to preserve old behavior?

Reply:
There's a Travis bug if waffle can't cache normally. My understanding of it is running waffle migrations before the cachetable migration means that waffle tries to cache in the DB table which doesn't exist yet. Here's an example: https://travis-ci.org/Johnetordoff/osf.io/jobs/427258977

Reply:
Oh ok. But is waffle caching in the same way (LocalMemCache) that it was before this change?

Reply:
Yep

Reply:
OK, great. 👌